youtube_dl/extractor/vtm.py

```diff
@@ -0,0 +1,125 @@
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from ..utils import (
+    urlencode_postdata,
+    compat_urllib_parse_urlencode,
+    ExtractorError,
+)
+
+
+class VTMIE(InfoExtractor):
+    """Download full episodes that require an account from vtm.be or q2.be.
+
+    The generic extractor can be used to download clips that do no require an
+    account.
+    """
+    _VALID_URL = r'https?://(?:www\.)?(?P<site_id>vtm|q2)\.be/video[/?].+?'
+    _NETRC_MACHINE = 'vtm'
+    _APIKEY = '3_HZ0FtkMW_gOyKlqQzW5_0FHRC7Nd5XpXJZcDdXY4pk5eES2ZWmejRW5egwVm4ug-'
+    _TESTS = [
+        {
+            'url': 'http://vtm.be/video/volledige-afleveringen/id/257085630173000',
+            'md5': 'ebd8bed45694f7f35ed19addee2717e8',
+            'info_dict': {
+                'id': '257085630173000',
+                'ext': 'mp4',
+                'title': 'Blind Getrouwd afl.1',
+                'description': 'md5:6526d6203a76bc572fe971211006bb70',
+            },
+            'skip': 'This video is only available for registered users'
+        }
+    ]
+
+    def _login(self):
+        (username, password) = self._get_login_info()
+        if username is None or password is None:
+            self.raise_login_required()
```

Comment:
No, because videos not requiring authentication are not matched by the URL regex as they don't have '/video' in the URL. So youtube-dl falls back to the generic extractor which works fine for those videos. I'll add a comment.